alien4cloud-yorc-plugin/src/main/java/org/ystia/yorc/alien4cloud/plugin/EventListenerTask.java

```diff
@@ -56,14 +56,24 @@ public void run() {
                     if (eventResponse.getEvents() != null) {
                         for (Event event : eventResponse.getEvents()) {
                             String paasId = event.getDeployment_id();
+                            log.debug("Received event from Yorc: " + event.toString());
+                            log.debug("Received event has deploymentId : " + paasId);
+                            String deploymentId = orchestrator.getDeploymentId(paasId);
+                            if (deploymentId == null) {
```

Comment:
This part is duplicated in the LogListener it may worth to factorize it if possible.